JoinColony/colonyCDapp
# Actions pagination

## Description

This PR adds pagination to colony actions list

<img width="724" alt="image" src="https://user-images.githubusercontent.com/112586815/217111085-43169c6f-07ec-4112-a0fb-7a97b27c62b5.png">

## Testing
1. Since you can only see actions of one type, apply the following patch to also display block numbers next to action items (it'll make it easier to verify sorting works as expected):
```
diff --git a/src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx b/src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx
index 6da2a0d3..67c0f38b 100644
--- a/src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx
+++ b/src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx
@@ -82,13 +82,14 @@ const ActionsListItem = ({
       }
       createdAt={createdAt}
       extra={
-        null // displayCountdownTimer && (
-        //   <CountdownTimer
-        //     state={motionState as MotionState}
-        //     motionId={Number(motionId)}
-        //     isFullyNayStaked={isFullyNayStaked(totalNayStake, requiredStake)}
-        //   />
-        // )
+        // null // displayCountdownTimer && (
+        // //   <CountdownTimer
+        // //     state={motionState as MotionState}
+        // //     motionId={Number(motionId)}
+        // //     isFullyNayStaked={isFullyNayStaked(totalNayStake, requiredStake)}
+        // //   />
+        // // )
+        <div>{item.blockNumber}</div>
       }
       meta={<ActionsListItemMeta fromDomainId={fromDomain ?? ''} />}
       onClick={handleActionRedirect}

```

2. Create a colony using the colony creation wizard (**NOTE**: Mint tokens will not work with a script-created colony)
3. Mint tokens at least 11 times.
5. Check that only 10 items are shown initially, in a correct order, the Load more buttons works and that you can change sorting options


Resolves #210
